AdyenActions/Components/3DS2/Action handlers/ThreeDS2ClassicActionHandler.swift

```diff
@@ -55,8 +55,18 @@ internal class ThreeDS2ClassicActionHandler: AnyThreeDS2ActionHandler, Component
     internal init(context: AdyenContext,
                   appearanceConfiguration: ADYAppearanceConfiguration) {
         self.context = context
-        self.coreActionHandler = ThreeDS2CoreActionHandler(context: context,
-                                                           appearanceConfiguration: appearanceConfiguration)
+        #if canImport(AdyenAuthentication)
+            if #available(iOS 14.0, *) {
+                self.coreActionHandler = ThreeDS2PlusDACoreActionHandler(context: context,
```

Comment:
Do you know when are we planing to deprecate "classic" 3DS with two actions?

Reply:
I don't think we can remove it, the classic flow is still possible with an ADP